source/includes/fact-non-utf8-data.rst

```diff
@@ -0,0 +1,17 @@
+If Compass encounters data display issues when querying your data 
+with the error message: ``Invalid UTF-8 string in BSON document``. You 
+can disable UTF8 validation by setting the ``enableUtf8Validation`` 
+URI option to ``false``. 
+
+The following URI disables UTF8 validation:
```

Comment:
(question)
Where do you set this, apart from the Advanced Connection Options? We should tell folks.

Reply:
These are the only two options, the URI or the adv connections tab to my knowledge.